generators/client/templates/client/source/main.jsx

```diff
@@ -20,16 +18,13 @@ class AppRoot extends React.Component {
     history: React.PropTypes.object.isRequired, // eslint-disable-line
   };
 
-  constructor(props) {
-    super(props);
-    this.routes = routes;
-  }
-
   render() {
     return (
       <Provider store={this.props.store}>
         <MuiThemeProvider muiTheme={muiTheme}>
-          <Router history={this.props.history} routes={this.routes} />
+          <BrowserRouter basename="/" >
```

Comment:
I'm not sure but `basename` should also depend on `applicationName` (`basename="/<%= applicationName %>/"`) to avoid 404 error when refreshing pages ?